src/Clock.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+type State = {
+  time: string;
+};
+
+type Props = {
+  clockName: number;
+};
+
+export class Clock extends React.Component<Props, State> {
+  state: State = {
+    time: new Date().toLocaleTimeString(),
+  };
+
+  update = setInterval(() => {
+    this.setState({ time: new Date().toLocaleTimeString() });
+
+    // eslint-disable-next-line
+    console.log(this.state.time);
```

Comment:
you can call toLocaleTimeString() only here and get rid of it in state and setState